docs/source/interactive.rst

```diff
@@ -5,99 +5,138 @@ Dask-jobqueue is most often used for interactive processing using tools like
+
+   # Install JupyterLab and NodeJS (which we'll need to integrate Dask into JLab)
+   conda install jupyterlab nodejs -c conda-forge -y
 
-It is convenient to run a Jupyter notebook server on the HPC for use with
-dask-jobqueue. You may already have a Jupyterhub instance available on your
-system, which can be used as is. Otherwise, documentation for starting your own
-Jupyter notebook server is available at `Pangeo documentation
-<http://pangeo-data.org/setup_guides/hpc.html#configure-jupyter>`_.
+   # Install server-side pieces of the Dask-JupyterLab extension
+   pip install dask_labextension
 
-Once Jupyter is installed and configured, using a Jupyter notebook is done by:
+   # Integrate Dask-Labextension with Jupyter (requires NodeJS)
+   jupyter labextension install dask-labextension
 
-- Starting a Jupyter notebook server on the HPC (it is often good practice to
-  run/submit this as a job to an interactive queue, see Pangeo docs for more
-  details).
+You can also use ``pip`` rather than ``conda``, but you will have to find some
+other way to install NodeJS.
+
+.. image:: https://github.com/dask/dask-labextension/raw/master/dask.png
+   :width: 50%
+   :alt: Dask JupyterLab Dashboard
+
+Add A Password
+--------------
+
+For security, we recommend adding a password to your Jupyter notebook
+configuration.
 
 .. code-block:: bash
 
-   $ jupyter notebook --no-browser --ip=`hostname` --port=8888
+   jupyter notebook password
 
-- Reading the output of the command above to get the ip or hostname of your
-  notebook, and use SSH tunneling on your local machine to access the notebook.
-  This must only be done in the probable case where you don't have direct
-  access to the notebook URL from your computer browser.
+This is good both for security, and also means that you won't have to copy
+around Jupyter tokens.
+
+
+Start Jupyter
+-------------
+
+When you use Jupyter on your laptop you often just write ``jupyter notebook``
+or ``jupyter lab``.  However, things are a bit different when starting a
+notebook server on a separate machine.  As a first step, the following will
+work:
 
 .. code-block:: bash
 
-   $ ssh -N -L 8888:x.x.x.x:8888 username@hpc_domain
+   jupyter lab --no-browser --ip="*" --port 8888
 
-Now you can go to ``http://localhost:8888`` on your browser to access the
-notebook server.
+Later, once we get SSH tunneling set up, you may want to come back and specify
+a specific IP address or hostname for added security.
 
 
-Viewing the Dask Dashboard
---------------------------
+SSH Tunneling
+-------------
+
+If your personal machine is on the same network as your cluster, then you can
+ignore this step.
+
+If you are on a different network (like your home network), and have to SSH in,
+then it can be difficult to have your local web browser connect to the Jupyter
+server running on the HPC machine.  If your institution doesn't have something
+like `JupyterHub <https://jupyter.org/hub>`_ set up, then the easiest way to
```

Comment:
Why not talking about Jupyterhub from the start? The same sentence or nearly in introduction, to say that you don't want to do any of this if a Jupyterhub is available (except if launched notebook are too old, or to be able to use dask-labextension if not installed, but that's some advanced business).

Reply:
I've added a JupyterHub advertisement at the top.